slf4j-jdk-platform-logging/src/main/java/org/slf4j/jdk/SLF4JSystemLogger.java

```diff
@@ -0,0 +1,136 @@
+package org.slf4j.jdk;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.MissingResourceException;
+import java.util.ResourceBundle;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Adapts {@link Logger} to {@link System.Logger}.
+ */
+class SLF4JSystemLogger implements System.Logger {
+
+    private static final Logger INTERNAL_LOGGER = LoggerFactory.getLogger(SLF4JSystemLogger.class);
+
+    private final Logger logger;
+
+    public SLF4JSystemLogger(Logger logger) {
+        this.logger = requireNonNull(logger);
+    }
+
+    @Override
+    public String getName() {
+        return logger.getName();
+    }
+
+    @Override
+    public boolean isLoggable(Level level) {
+        switch(level) {
+            case ALL:
+                // fall-through intended because `ALL` is loggable if the
+                // lowest level is enabled
+            case TRACE:
+                return logger.isTraceEnabled();
+            case DEBUG:
+                return logger.isDebugEnabled();
+            case INFO:
+                return logger.isInfoEnabled();
+            case WARNING:
+                return logger.isWarnEnabled();
+            case ERROR:
+                return logger.isErrorEnabled();
+            case OFF:
+                // all logging is disabled if the highest level is disabled
+                return !logger.isErrorEnabled();
+            default:
+                INTERNAL_LOGGER.error(
+                        "SLF4J internal error: unknown log level {} passed to `isLoggable` (likely by the JDK).", level);
+                return true;
```

Comment:
I was unsure how to such handle error cases. They're highly unlikely to occur (JDK API should be very stable), but if they do, IMHO SLF4J should not crash, misbehave as little as possible, but let the user know that there's a problem (hence the error logger). See below for two more such cases.

Reply:
To avoid error cases from future JDK changes, you could compare to `Level.getSeverity()`.
Basic idea is that you don't want to map all j.u.l. levels, you want to start from the available SLF4J levels and determine into which bracket the passed-in j.u.l. level fits. (FINER/FINEST etc. have different severities, despite being mapped to the same j.u.l. level.)